test/controllers/IndexControllerSpec.scala

```diff
@@ -18,278 +18,162 @@ package controllers
-            bind[UserAnswerService].toInstance(mockUserAnswerService),
-            bind[CharitiesShortLivedCache].toInstance(mockCharitiesShortLivedCache),
-            bind[AuthIdentifierAction].to[FakeAuthIdentifierAction]
-          )
-          .build()
-
-      val controller: IndexController = app.injector.instanceOf[IndexController]
-
-      when(mockCharitiesShortLivedCache.fetchAndGetEntry[Boolean](any(), any())(any(), any(), any()))
-        .thenReturn(Future.successful(None))
-      when(mockUserAnswerService.get(any())(any(), any())).thenReturn(
-        Future.successful(
-          Some(
-            emptyUserAnswers
-              .set(OldServiceSubmissionPage, OldServiceSubmission("num", "date"))
-              .success
-              .value
-          )
-        )
-      )
-      when(mockUserAnswerService.set(any())(any(), any())).thenReturn(Future.successful(true))
+        when(mockUserAnswerService.get(any())(any(), any())).thenReturn(Future(None))
 
-      lazy val result = controller.onPageLoad()(fakeRequest)
+        lazy val result = controller.onPageLoad()(fakeRequest)
 
-      status(result) mustEqual SEE_OTHER
-      redirectLocation(result).value mustEqual routes.ApplicationBeingProcessedController.onPageLoad.url
-      verify(mockUserAnswerService, times(1)).get(any())(any(), any())
+        status(result) mustEqual 303
+        redirectLocation(result).value mustBe controllers.routes.CannotFindApplicationController.onPageLoad.url
+        verify(mockUserAnswerService, times(1)).get(any())(any(), any())
+
+      }
     }
 
-    "Set answers and redirect to the next page (start of journey page)" in {
+    "there are  empty useranswers and a session id" must {
 
-      when(mockUserAnswerService.get(any())(any(), any())).thenReturn(Future.successful(None))
-      when(mockCharitiesShortLivedCache.fetchAndGetEntry[Boolean](any(), any())(any(), any(), any()))
-        .thenReturn(Future.successful(Some(true)))
-      when(mockUserAnswerService.set(any())(any(), any())).thenReturn(Future.successful(true))
+      "redirect to the task-list page" in {
 
-      val result = controller.onPageLoad()(fakeRequest)
+        when(mockUserAnswerService.get(any())(any(), any())).thenReturn(Future(Some(emptyUserAnswers)))
+        when(mockUserAnswerService.set(any())(any(), any())).thenReturn(Future(true))
 
-      status(result) mustEqual OK
-      verify(mockUserAnswerService, times(1)).get(any())(any(), any())
+        val result = controller.onPageLoad()(fakeRequest)
+
+        status(result) mustEqual OK
+        titleOf(contentAsString(result)) mustBe
+          "Add information about the charity - Register your charity’s details with HMRC - GOV.UK"
+        verify(mockUserAnswerService, times(1)).get(any())(any(), any())
+      }
     }
 
-    "Fetch user answers and redirect to the next page (start of journey page)" in {
+    "Fetch user answers and redirect to the task-list page" in {
```

Comment:
This should also follow the when -> must -> in. Currently it is when -> in which doesn't read well.